analytics/models/java.py

```diff
@@ -0,0 +1,25 @@
+from analytics.models.models import SubmissionAnalytics
+from django.db import models
+from course.fields import JSONField
+
+
+class JavaSubmissionAnalytics(SubmissionAnalytics):
+    lines = models.IntegerField(default=0)
+    blank_lines = models.IntegerField(default=0)
+    comment_lines = models.IntegerField(default=0)
+    import_lines = models.IntegerField(default=0)
+    cc = models.IntegerField(default=0)
```

Comment:
We need to have better names. All the names should be meaningful. `cc` is not a good name, if it's an abbreviation please write it in expanded form.

Reply:
Fixed.